src/main/java/sh/platform/config/reader/Config.java

```diff
@@ -39,6 +45,14 @@
 
     }
 
+    /**
+     * Creates a new instance with the environments sets.
+     * By default, it will read the values and keep the {@link PlatformVariables}. Otherwise, it will the fallback code.
```

Comment:
The second sentence here is a grammar parse error.  What's the intent?

Reply:
> Creates a new instance from the environments values that come from {@link PlatformVariables}
Better?